Server/src/main/java/gotcha/server/Service/API/ApiController.java

```diff
@@ -289,7 +296,7 @@ public Response view_admins(@SessionAttribute("userContext") UserContext userCon
     @RequestMapping(value = "/add_admin")
     @CrossOrigin
     @Override
-    public Response add_admin(String user_email,String name, String lastName, String user_password, String phoneNumber, LocalDate birthDay, String gender, @SessionAttribute("userContext") UserContext userContext) {
+    public Response add_admin(String user_email,String name, String lastName, String user_password, String phoneNumber, String birthDay, String gender, @SessionAttribute("userContext") UserContext userContext) {
```

Comment:
why not LocalDate?

Reply:
i had troubles to transfer LocalDate, i decide that the server will build the Local Date for now.